src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,35 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+interface Props {
+  selectedTodo: Todo;
+  handleClose: () => void;
+}
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, handleClose }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  const loadUser = async () => {
+    const selectedUser = await getUser(selectedTodo.userId);
+
+    setUser(selectedUser);
+  };
+
+  useEffect(
+    () => {
+      loadUser();
+    },
+    [],
```

Comment:
The useEffect dependency array is empty, which means the effect will only run once after the initial render. However, you may want to re-run the effect whenever the selectedTodo changes.